### PR TITLE
PMM-3920 MongoDB command.

### PR DESCRIPTION
**Pull request by idexter**

### CODE REVIEW

File: commands/management/add_mongodb.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	"strconv"
+
+	"github.com/percona/pmm/api/managementpb/json/client"
+	mongodb "github.com/percona/pmm/api/managementpb/json/client/mongo_db"
+
+	"github.com/percona/pmm-admin/agentlocal"
+	"github.com/percona/pmm-admin/commands"
+)
+
+var addMongoDbResultT = commands.ParseTemplate(`
```
Comment (AlekSi) [inline commands/management/add_mongodb.go]: `addMongoDBResultT`

File: commands/management/add_mongodb.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	AddMongoDBC.Arg("address", "MongoDB address and port. Default: 127.0.0.1:27017.").Default("127.0.0.1:27017").StringVar(&AddMongoDB.AddressPort)
+
+	hostname, _ := os.Hostname()
+	serviceName := hostname + "-mongodb"
+	serviceNameHelp := fmt.Sprintf("Service name. Default: %s.", serviceName)
+	AddMongoDBC.Arg("name", serviceNameHelp).Default(serviceName).StringVar(&AddMongoDB.ServiceName)
+
+	AddMongoDBC.Flag("username", "MySQL username.").StringVar(&AddMongoDB.Username)
+	AddMongoDBC.Flag("password", "MySQL password.").StringVar(&AddMongoDB.Password)
+	AddMongoDBC.Flag("use-profiler", "Run QAN profiler agent.").BoolVar(&AddMongoDB.UseProfiler)
```
Comment (AlekSi) [inline commands/management/add_mongodb.go:114]: We need a similar flag for mongodb_exporter.

File: commands/management/add_mongodb.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+func init() {
+	AddMongoDBC.Arg("address", "MongoDB address and port. Default: 127.0.0.1:27017.").Default("127.0.0.1:27017").StringVar(&AddMongoDB.AddressPort)
+
+	hostname, _ := os.Hostname()
+	serviceName := hostname + "-mongodb"
+	serviceNameHelp := fmt.Sprintf("Service name. Default: %s.", serviceName)
+	AddMongoDBC.Arg("name", serviceNameHelp).Default(serviceName).StringVar(&AddMongoDB.ServiceName)
+
+	AddMongoDBC.Flag("username", "MySQL username.").StringVar(&AddMongoDB.Username)
+	AddMongoDBC.Flag("password", "MySQL password.").StringVar(&AddMongoDB.Password)
```
Comment (AlekSi) [inline commands/management/add_mongodb.go]: Not MySQL.

File: commands/management/add_mysql.go [hunk 4]
```diff
@@ -88,7 +88,7 @@ func (cmd *addMySQLCommand) Run() (commands.Result, error) {
 		},
 		Context: commands.Ctx,
 	}
-	resp, err := client.Default.MySQL.Add(params)
+	resp, err := client.Default.MySQL.AddMixin1(params)
```
Comment (AlekSi) [inline commands/management/add_mysql.go]: That's not okay.

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: commands/management/add_mongodb.go (hunk 2 above)
Comment (idexter) [reply commands/management/add_mongodb.go:114]: Did you mean we need the same flag for having a possibility to enable/disable mongodb_exporter when using this command? If yes, I have added it.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**